apps/judicial-system/web/src/routes/CourtOfAppeal/components/CaseOverviewHeader/CaseOverviewHeader.strings.ts

```diff
@@ -1,11 +1,19 @@
 import { defineMessages } from 'react-intl'
 
 export const courtOfAppealCaseOverviewHeader = defineMessages({
-  appealedInfo: {
-    id: 'judicial.system.core:court_of_appeal_case_overview_header.appealed_by',
+  appealedBy: {
+    id: 'judicial.system.core:court_of_appeal.case_overview_header.appealed_by',
     defaultMessage:
       'Kært af {appealedByProsecutor, select, true {sækjanda} other {verjanda}} {appealedDate}',
     description:
       'Notaður sem texti fyrir "Kært af" á yfirlitsskjá mála hjá Landsrétti.',
   },
+  appealedByInCourt: {
+    id:
+      'judicial.system.core:court_of_appeal.case_overview_header.appealed_by_in_court',
+    defaultMessage:
+      '{appealedByProsecutor, select, true {Sóknaraðili} other {Varnaraðili}} kærði í þinghaldi',
```

Comment:
In the other string we say sækjandi and here we say sóknaraðili (and verjanda vs. varnaraðila), why do we use different titles for the same person?